zerver/views/webhooks/trello/board_actions.py

```diff
@@ -0,0 +1,107 @@
+from typing import Dict, Any, Tuple
+from .exceptions import UnknownUpdateBoardAction
+from .templates import TRELLO_SUBJECT_TEMPLATE, TRELLO_MESSAGE_TEMPLATE
+
+SUPPORTED_BOARD_ACTIONS = [
+    'removeMemberFromBoard',
+    'addMemberToBoard',
+    'createList',
+    'updateBoard',
+]
+
+REMOVE_MEMBER = 'removeMemberFromBoard'
+ADD_MEMBER = 'addMemberToBoard'
+CREATE_LIST = 'createList'
+CHANGE_NAME = 'changeName'
+
+TRELLO_BOARD_URL_TEMPLATE = '[{board_name}]({board_url})'
```

Comment:
All message content should be `text_type`. So prefix this string with a `u` to make it a unicode string.